<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshForm</class>
 <widget class="QWidget" name="MeshForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>140</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="transHorizontalWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="meshUpperWidget" native="true">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignLeft|Qt::AlignTop">
            <widget class="QLabel" name="MeshComponentLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Mesh Component</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="meshLowerWidget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="translateLayout">
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QLabel" name="MeshLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Current Mesh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="meshCompoBox"/>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QCheckBox" name="checkBox">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>IsDrawingTriangles</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
